spyder/plugins/ipythonconsole/widgets/shell.py

```diff
@@ -161,7 +161,7 @@ def will_close(self, externally_managed):
         super(ShellWidget, self).will_close(externally_managed)
 
     def call_kernel(self, interrupt=False, blocking=False, callback=None,
-                    timeout=None):
+                    timeout=None, display_error=False):
```

Comment:
Please expand the docstring for this function explaining what this new `display_error` kwarg does.